doc/rtd/howto/predeploy_testing.rst

```diff
@@ -3,139 +3,69 @@
 How to test ``cloud-init`` locally before deploying
 ***************************************************
 
+How to run cloud-init locally
+=============================
 It's very likely that you will want to test ``cloud-init`` locally before
 deploying it to the cloud. Fortunately, there are several different virtual
-machines (VMs) and container tools that are ideal for this sort of local
+machine (VM) and container tools that are ideal for this sort of local
 testing.
 
-In this guide, we will show how to use three of the most popular tools:
-`Multipass`_, `LXD`_ and `QEMU`_.
+To run cloud-init locally, head over to the
+:ref:`tutorials page<tutorial_index>` page, which will show you how to run
+cloud-init locally. Once you know how to run cloud-init locally, you may wish
+to "rerun" cloud-init while developing your cloud config for faster
+development.
 
-Multipass
-=========
+.. _how_to_rerun_cloud-init:
```

Comment:
I don't think tutorials should replace the multipass, lxd, and qemu sections here. Tutorials are for learning. How-tos are for getting things done. If I want to quickly spin up an LXD instance with cloud-init data, I'm not going to want to walk through a tutorial to figure out how to do it.

Reply:
That's fair. However, I am also going to replace the cloud-localds command with genisoimage - it has broader adoption than `cloud-utils`, and cloud-localds is a thin wrapper around it.